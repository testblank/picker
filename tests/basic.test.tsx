import { render } from 'enzyme';
import { html } from 'cheerio';
import * as React from 'react';
import Picker from '../src';

describe('basic', () => {
  it('classnames', () => {
    const wrapper = render(
      <Picker noAnimate={true}>
        <Picker.Item value="1">a</Picker.Item>
        <Picker.Item value="2">b</Picker.Item>
      </Picker>
    );

    expect(html(wrapper)).toMatchSnapshot();

    expect(wrapper.hasClass('rmc-picker')).toBeTruthy();
  });
});
