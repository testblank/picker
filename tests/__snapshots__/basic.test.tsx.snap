// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic classnames 1`] = `"<div class=\\"rmc-picker\\"><div class=\\"rmc-picker-mask\\"></div><div class=\\"rmc-picker-indicator \\"></div><div class=\\"rmc-picker-content\\"><div class=\\"rmc-picker-item rmc-picker-item-selected \\">a</div><div class=\\"rmc-picker-item \\">b</div></div></div>"`;
